metadata.json

```diff
@@ -1,6 +1,6 @@
 {
     "shell-version": [
-        "3.38"
+        "40.0"
```

Comment:
Note that `40.0` in the new version scheme is equivalent to `3.38.0` in the old one, i.e. this will stop working as soon as `40.1` is released in a couple of weeks.

Reply:
`40` is now a supported version on e.g.o website.